flink-contrib/flink-storm-examples/src/test/java/org/apache/flink/storm/split/SplitBoltTopology.java

```diff
@@ -28,7 +28,7 @@
 import org.apache.storm.topology.TopologyBuilder;
 
 /**
- * A simple topology that splits a numbers stream based the numbers parity, and verifies the result.
+ * A simple topology that splits a numbers stream based their parity, and verifies the result.
```

Comment:
This would be a "number stream" or a "stream of numbers"? And I think "based on ...".